shell/packages/sandstorm-db/migrations.js

```diff
@@ -784,6 +821,9 @@ function migrateToLatest() {
 
       Migrations.update({ _id: "new_server_migrations_applied" }, { $set: { value: true } });
     }
+
+    // Start background migrations.
+    backgroundFillInGrainSizes();
```

Comment:
We have other migrations that use `globalBackend`. I felt the code made the most sense here because it is a migration. Arguably migrations.js should be moved somewhere where it can more easily depend on other APIs since it commonly needs them.